elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/scaladsl/ElasticsearchFlow.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.elasticsearch.scaladsl
+
+import akka.NotUsed
+import akka.stream.alpakka.elasticsearch.{ElasticsearchFlowStage, ElasticsearchSinkSettings, IncomingMessage}
+import akka.stream.scaladsl.Flow
+import org.elasticsearch.client.{Response, RestClient}
+import spray.json.{DefaultJsonProtocol, JsObject, JsonWriter}
+
+object ElasticsearchFlow {
+
+  /**
```

Comment:
Reading the source code of the GraphStage, I see that it would be one `Response` per batch. Would there be anything you could do when a batch fails? Could it make sense to simplify and just make the GraphStage a sink that fails the flow when the request fails? Or maybe configure it with an exception handler (similar to a supervisor) that knows what to do when a single batch request fails?